app/helpers/accessible_format_helper.rb

```diff
@@ -0,0 +1,24 @@
+module AccessibleFormatHelper
```

Comment:
Hmm IMO we shouldn't really be forfeiting unit tests because a higher level tests touches the code, as they generally are not granular enough to thoroughly test the methods. However, if the request test actually covers all scenarios then I'm happy to leave the test out in this instance 👍

Reply:
Yeah, agreed re unit tests but here I think we genuinely are covering properly and it's the same argument to some extent as moving away from controller tests. If we cover adequately in the request tests and this is used nowhere else, not such a problem because we've tested it in the actual stack it's being used in. (Plus if I do simplify the radio items by moving them into the view this helper gets super light, those are the only methods worth testing really).

As an interesting experiment, though, I might make a test branch this and replace simplecov with DeepCover, which can deal with branch coverage, see what we get.

Reply:
DeepCover does indeed give us 100% branch coverage for this, 3 out of 3 branches covered completely. Interestingly not quite 100% node coverage! 1 node is never run (accessible_format_options_to_radio_items is never called without a parameter, so the default value for checked is unnecessary).